### PR TITLE
A few fixes

### LINKED ISSUE #146: SymPy output not interactive? (Julia 0.5)
If I do the following (using a nice example I saw introducing interact)

```
using SymPy

x=Sym("x")
@manipulate for n=1:50
    SymPy.diff(sin(x^2),x,n)
end
```

The rendering doesn't work, except for the default output (at the middle of the slider), which correctly renders (mathJax?) output into a nicely formatted equation. If I move the slider, the SymPy.diff command seems to still be processed correctly, but now the output is (say for n=1):

`$$2 x \cos{\left (x^{2} \right )}$$`

Any suggestions as to how to fully enable interactive output and have it render?

Many thanks

```
julia> versioninfo()
Julia Version 0.5.0
Commit 3c9d753 (2016-09-19 18:14 UTC)
Platform Info:
  System: Darwin (x86_64-apple-darwin13.4.0)
  CPU: Intel(R) Core(TM) M-5Y51 CPU @ 1.10GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.7.1 (ORCJIT, broadwell)
```

### PR DESCRIPTION
**Pull request by JobJob**
A few different fixes, see the commit messages for more details.

Also adds `vslider` and `vselection_slider`

I think it would be good to tag another release with these, the first commit is especially important since a `@manipulate` slider can behave weirdly without it - especially if the kernel is remote.

Commit 6 fixes #146 and the minor issue remaining in #144 (with the use of LaTeXStrings.jl)

### CODE REVIEW

File: src/IJulia/ijulia.js [hunk 1]
```diff
@@ -24,16 +24,19 @@
             var _src = 'data:' + type + ';base64,' + val;
 	    $(container).find("img").attr('src', _src);
 	    break;
+	case "text/latex":
+		if (MathJax){
+			var math = MathJax.Hub.getAllJax(container)[0];
```
Comment (shashi) [inline src/IJulia/ijulia.js]: what does this do?

Comment (JobJob) [inline src/IJulia/ijulia.js]: Gets the MathJax elements inside the output block: https://docs.mathjax.org/en/v2.5-latest/typeset.html#manipulating-individual-math-elements

Slightly paraphrasing: This looks up the list of math elements in the [container] element (I'm assuming there is only one) and takes the first one (element 0) and stores it in math. This is an element jax object (see the Element Jax specification for details), which has a Text() method that can be used to set the input text of the math element, and retypeset it.

Comment (shashi) [inline src/IJulia/ijulia.js]: Okay, thanks for the explanation, sounds good.

**Comment by shashi** [comment]
Feel free to merge it when you think this is ready.

**Comment by JobJob** [comment]
I'm ready, thanks for the review man.

**Comment by JobJob** [comment]
Do you think I should squash all these?

**Comment by shashi** [comment]
No, leave them. But select "Rebase and merge"

**Comment by JobJob** [comment]
Do you want to tag a new release? What's up with the `Pkg.update("IJulia")` stuff?

**Comment by shashi** [comment]
Guess we'll have to remove that and print a message instead.

**Comment by shashi** [comment]
I've got [attobot](https://github.com/attobot/attobot) set up. So you can make a release from the Github UI itself now.

**Comment by JobJob** [comment]
What happens if you just change the IJulia version in the REQUIRE

**Comment by shashi** [comment]
We don't have an IJulia version in REQUIRE :ghost:

That's because GtkInteract.jl doesn't need IJulia for example.

**Comment by JobJob** [comment]
Ahh ok :)

**Comment by tkelman** [comment]
> "Rebase and merge"

There's rarely any reason to do this option. Annoyingly github doesn't put a PR reference in the commit message when you do this.

**Pull request merged**